demos/_unit-test/post.php

```diff
@@ -10,6 +10,14 @@
 /** @var \Atk4\Ui\App $app */
 require_once __DIR__ . '/../init-app.php';
 
+// Creation of request from globals depenmds on getallheaders
```

Comment:
no hacks in individual files, this must be fixed in central place

if the comment about `getallheaders` is true, then https://github.com/Nyholm/psr7-server/blob/1.0.2/src/ServerRequestCreator.php#L52 needs to be replaced

Reply:
i move it to demos/init-app.php

i don't think the internal creation of ServerRequest must be changed, we need it only in unit tests running via CLI

Reply:
discussed in https://github.com/atk4/ui/pull/2101#discussion_r1321152445